pkg/resources/apiserver/deployment.go

```diff
@@ -643,22 +651,20 @@ func GetEnvVars(data kubeAPIServerEnvData) ([]corev1.EnvVar, error) {
 	}
 	cluster := data.Cluster()
 
-	var vars []corev1.EnvVar
+	vars := []corev1.EnvVar{
+		// Kubernetes <1.19 did not ship any certificates in their Docker images,
```

Comment:
It should be

> Kubernetes >1.19

or maybe

> Kubernetes 1.19+

Reply:
>1.19 is the opposite of <1.19 and from my tests, 1.17 and 1.18 failed because they were suddenly missing CAs, but with 1.19 they started shipping CAs.

Reply:
yeah, sorry for confusion 😅